include/md5_mb.h

```diff
@@ -206,7 +206,7 @@ MD5_HASH_CTX* md5_ctx_mgr_submit_sse (MD5_HASH_CTX_MGR* mgr, MD5_HASH_CTX* ctx,
 				const void* buffer, uint32_t len, HASH_CTX_FLAG flags);
 
 /**
- * @brief Finish all submitted MD5 jobs and return when complete.
+ * @brief Wait for any of submitted MD5 jobs to complete and return its ctx info when complete.
```

Comment:
"wait" is not the right word. 

It finish the min remain length job and return it .

Reply:
You are right - "wait" is wrong as this function actually does a work related to computing MD5.

Could you precise what you mean by "min remaining job length"? Is it part of work related to md5 previously submitted CTX-es? May submit return CTX which is not finished yet (e.g. in HASH_CTX_STS_PROCESSING state)?

Reply:
Multi buffer process multi-job at once . And submit will execute the data process if all lanes are busy. Inside the function, It will process data length with minimum size of jobs, then a job will be finished  and it's ctx will be return .  It always return the minimum size job. For the next submit call , other jobs length is remaining length.  That's my meaning of  "min remaining job length" . 
That's not important in comments. That's a inside code logic.

>>Is it part of work related to md5 previously submitted CTX-es? 
Yes. all code follow the similar flow. 

>>May submit return CTX which is not finished yet (e.g. in HASH_CTX_STS_PROCESSING state)?
No. It only return finished job.

Reply:
I agree "wait" is not quite right.  Perhaps "Finish **a** submitted MD5 job if any outstanding."